src/SelfUpdateCommand.php

```diff
@@ -25,6 +25,26 @@ class SelfUpdateCommand extends Command
 
     protected $applicationName;
 
+    /**
+     * @var bool
+     */
+    protected $isPreview = false;
+
+    /**
+     * @var bool
+     */
+    protected $isStable = true;
+
+    /**
+     * @var bool
+     */
+    protected $isCompatible = false;
```

Comment:
Naming things is hard: `isCompatible` is hard to understand because there is no comment, takes some reading to see that this is the "compatible" option. `$isCompatibleOptionSelected`? Optional.

Reply:
Fixed